test/jdk/jdk/internal/vm/TestTranslatedException.java

```diff
@@ -58,15 +58,6 @@ public void encodeDecodeTest() throws Exception {
         }
         encodeDecode(throwable);
     }
-    @SuppressWarnings("unchecked")
-    @Test
-    public void encodeDecodeTest2() throws Exception {
-        Throwable throwable = new ExceptionInInitializerError(new InvocationTargetException(new Untranslatable("test exception", new NullPointerException()), "invoke"));
-        for (int i = 0; i < 10; i++) {
-            throwable = new ExceptionInInitializerError(new InvocationTargetException(new RuntimeException(String.valueOf(i), throwable), "invoke"));
-        }
-        encodeDecode(throwable);
-    }
 
```

Comment:
Why this was removed?

Reply:
Because it does exactly the same thing as `encodeDecodeTest`. It should have been cleaned up in the original PR that introduced this test.